lib/memo/it.rb

```diff
@@ -1,13 +1,18 @@
 module Memo
   VERSION = '0.2.0'
   module It
-    def memo(ignore: [], &block)
-      keys = block.source_location
-      ignore = Array(ignore)
-      keys << block.binding.local_variables.map do |name|
-        next if ignore.include?(name)
-        [name, block.binding.local_variable_get(name)]
+    def memo(only: [], ignore: [], &block)
```

Comment:
Nice. Just wondering if we should rename `ignore` to `except`, as it's the naming pattern found in Rails & Co.